ddtrace/contrib/bottle/trace.py

```diff
@@ -81,6 +81,8 @@ def wrapped(*args, **kwargs):
                         url=url,
                         status_code=response_code,
                         query=request.query_string,
+                        request_headers=request.headers,
+                        response_headers=response.headers,
```

Comment:
Same as with aiohttp, we should probably have a test for response headers as well